tsconfig.json

```diff
@@ -35,6 +35,7 @@
     "src/analysis/retail/paladin/retribution",
     "src/analysis/retail/priest/shadow",
     "src/analysis/retail/rogue/assassination",
+    "src/analysis/retail/shaman/shared",
```

Comment:
I remember removing this line in the tsconfig. Are you running into issues with the modules in this folder?

Reply:
I had to merge the origin to my local.  This was a conflict I got.

Reply:
@djanse did you want me to remove it again?

Reply:
@jsucupira That would be preferable assuming it doesnt result into any errors.

Reply:
@djanse it shouldn't, it was just a conflict, I am guessing I didn't solve it correctly.

Reply:
If it does, just leave it as is and i'll go over it again :)

Reply:
@djanse it built without an issue. 
![image](https://user-images.githubusercontent.com/1004621/200078400-3da3456c-97a2-42a1-957f-132b18fc7955.png)